mkdocs.yml

```diff
@@ -120,36 +120,36 @@ nav:
       - 'Getting started' : 'user-guide_overview.md'
       - 'Optimised software list' : 'dietpi_optimised_software.md'
-      - 'Supported SBCs' : 'dietpi_sbc.md'
-      - 'Community tutorials' : 'https://dietpi.com/phpbb/viewforum.php?f=15'
       - 'Advanced configuration' : 'dietpi_tools.md'
-      - 'Troubleshooting' : 'https://dietpi.com/phpbb/viewforum.php?f=11'
+      - 'Supported SBCs' : 'dietpi_sbc.md'
       - 'Release notes' : 'release-notes.md'
       - 'Request features' : 'https://github.com/MichaIng/DietPi/issues/new/choose'
       - 'Report issues' : 'https://github.com/MichaIng/DietPi/issues'
       - 'Documentation tasks' : 'dietpi_docs_todo.md'
   - 'OPTIMISED SOFTWARE' :
+      - 'BitTorrent & Download Tools' : 'software/bittorrent.md'
+      - 'Camera & Surveillance' : 'software/camera.md'
+      - 'Cloud & Backup systems' : 'software/cloud.md'
+      - 'Computational Science' : 'software/computational_science.md'
       - 'Desktops & Utilities' : 'software/desktop.md'
+      - 'DNS Server' : 'software/dns_servers.md'
+      - 'File Server' : 'software/file_servers.md'
+      - 'Gaming & Emulation' : 'software/gaming.md'
+      - 'Hardware Projects' : 'software/hardware_projects.md'
+      - 'Home Automation' : 'software/home_automation.md'
+      - 'Logging Systems' : 'software/log_system.md'
       - 'Media Systems': 'software/media.md'
+      - 'Network Load Balancer' : 'software/load_balancer.md'
+      - 'Printing Server' : 'software/printing.md'
       - 'Remote Desktop & Remote Access' : 'software/remote_desktop.md'
-      - 'Cloud & Backups' : 'software/cloud.md'
-      - 'BitTorrent & Download Tools' : 'software/bittorrent.md'
-      - 'Gaming & Emulation' : 'software/gaming.md'
-      - 'Camera & Surveillance' : 'software/camera.md'
       - 'Social & Search' : 'software/social.md'
-      - 'WiFi Hotspot' : 'software/wifi_hotspot.md'
-      - 'Home Automation' : 'software/home_automation.md'
-      - 'Hardware Projects' : 'software/hardware_projects.md'
-      - 'System Stats & Management' : 'software/system_stats.md'
+      - 'SSH Server' : 'software/ssh.md'
       - 'System Security' : 'software/system_security.md'
-      - 'Web Development' : 'software/webserver_stack.md'
-      - 'DNS Servers' : 'software/dns_servers.md'
-      - 'File Servers' : 'software/file_servers.md'
+      - 'System Stats & Management' : 'software/system_stats.md'
       - 'VPN' : 'software/vpn.md'
-      - 'Network Load Balancer' : 'software/load_balancer.md'
-      - 'Printing Servers' : 'software/printing.md'
-      - 'Computational Science' : 'software/computational_science.md'
-      - 'SSH Servers' : 'software/ssh.md'
-      - 'Logging Systems' : 'software/log_system.md'
+      - 'Web Development' : 'software/webserver_stack.md'
+      - 'WiFi Hotspot' : 'software/wifi_hotspot.md'
   - 'COMMUNITY' :
       - 'DietPi Forums & Community' : 'https://dietpi.com/phpbb/viewforum.php?f=5'
```

Comment:
Maybe remove the `& Community` here? The word doubles a bid with the navigation tab name and the "Community tutorials" below.

Reply:
Ok.